src/main/scala/com/danielasfregola/twitter4s/entities/User.scala

```diff
@@ -33,8 +33,8 @@ final case class User(blocked_by: Boolean = false,
                       profile_background_image_url_https: String,
                       profile_background_tile: Boolean = false,
                       profile_banner_url: Option[String] = None,
-                      profile_image_url: ProfileImage,
-                      profile_image_url_https: ProfileImage,
+                      profile_image_url: Option[ProfileImage] = None,
```

Comment:
Why the defaults? I don't think we need them

Reply:
removed